docs/solana/07-governance.md

```diff
@@ -4,8 +4,14 @@ sidebar_position: 7
 title: "Governance"
 ---
 
-The Protocol Admin is an account with exclusive access to collecting the fee from the treasury account, and also has the
-"upgrade authority". More concretely, the Admin is a multisig wallet and currently in control of Sablier Labs.
+SolSab program governance comprises two key functions:
+
+- Collecting accumulated protocol fees from the Treasury account and
```

Comment:
Since these are in the form of lists, may be we can remove `and` from the end of the sentence. Wdyt?